packages/fluentui/react-northstar/src/themes/teams/components/Datepicker/datepickerVariables.ts

```diff
@@ -29,9 +34,11 @@ export interface DatepickerVariables {
 }
 
 export const datepickerVariables = (siteVars): DatepickerVariables => ({
+  calendarGridWidth: pxToRem(DAYS_IN_WEEK * 30),
+
   calendarCellBorder: 'none',
-  calendarCellHeight: pxToRem(32),
-  calendarCellWidth: pxToRem(32),
+  calendarCellHeight: pxToRem(30),
```

Comment:
Please why did you change the height from 32px to 30px?

Reply:
Fixed layout :)